docs/documentation/Logs.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Storm Logs
+layout: documentation
+documentation: true
+---
+Logs in Storm are essential for tracking the status, operations, error messages and debug information for all the 
+daemons (nimbus, supervisor, logviewer, drpc, ui) and topologies' workers.
+
+### Location of the Logs
+All the daemon logs are placed under ${storm.log.dir} directory, which user can set in System property or
```

Comment:
Grammar. 
which user can set in System property  --> which "a" user can set in "the" System property